src/Common/AssemblyLoadingUtils.cs

```diff
@@ -140,9 +140,12 @@ private static bool ShouldSkipPath(string path)
             string name = Path.GetFileName(path).ToLowerInvariant();
             switch (name)
             {
+                case "cpumathnative.dll":
```

Comment:
>cpumathnative [](start = 22, length = 13)

symsgdnative.dll as well.

how it works for linux btw? we have libCpuMathNative.so not cpumathnative.dll

Reply:
I'll add symsgdnative.dll as well.

It works for linux because we are enumerating `*.dll` and then calling `ShouldSkipPath` on the results of that.  On linux, the file extension is `.so`, so that doesn't match `*.dll`, which means those assemblies are never returned.